aio/content/examples/schematics-for-libraries/projects/my-lib/package.json

```
@@ -0,0 +1,20 @@
+// #docplaster
+// #docregion collection
+{
+  "name": "my-lib",
+  "version": "0.0.1",
+// #enddocregion collection
+  "scripts": {
+    "build": "../../node_modules/.bin/tsc -p tsconfig.schematics.json",
+    "copy:schemas": "cp --parents schematics/*/schema.json ../../dist/my-lib/",
+    "copy:files": "cp --parents -p schematics/*/files/** ../../dist/my-lib/",
```

`--parents` is a non-POSIX flag of cp and is not supported in all systems.

See this [Superuser](https://superuser.com/q/684643/636559) question for more info.

---

Here's a code suggestion:
```suggestion
    "copy:files": "find schematics/*/files/** -exec rsync -R {} ../../dist/ngx-ytd-api-lib/ \\;",
```

---

This will be resolved in PR #29505